libraries/Matter/examples/MatterDimmableLight/MatterDimmableLight.ino

```diff
@@ -0,0 +1,157 @@
+// Matter Manager
```

Comment:
Could you please add a brief description in the sketch of how the example works ?

Reply:
I'll prepare a whole documentation latter when I add a few more endpoints.